### PR TITLE
Clarify documentation for creating new test suites.

### PR DESCRIPTION
**Pull request by owenlittlejohns**
## Description

This PR adds some missing information to instruct people the changes they need to make when creating a new test suite.

## Jira Issue ID

N/A

## Local Test Steps

N/A

## PR Acceptance Checklist
* ~~Acceptance criteria met~~
* ~~Tests added/updated (if needed) and passing~~
* [x] Documentation updated (if needed)~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,11 @@ created Zarr store.*
 
 1. *`HARMONY_HOST_URL` is the harmony base url for your target environment. e.g. `SIT` would be `https://harmony.sit.earthdata.nasa.gov`*
 
+1. When testing against a local Harmony installation, where
+  `HARMONY_HOST_URL=http://localhost:3000`, best testing results occur when
+  running the notebook manually on a Jupyter notebook server, rather than via
+  the `run_notebooks.sh` script.
```
Comment (flamingbear) [inline README.md:61]: I'm pretty sure you can't run the run_notebooks.sh script against local.  docker in docker problem?

File: README.md [hunk 2]
```diff
@@ -74,6 +79,48 @@ environment before installing from the environment.yml.
...
+   images: <pre existing rules already listed> <new-suite-name>-image
+   ```
+1. Update `test/run_notebooks.sh` to include the new test image in `all_images`:
+   ```
+   all_images=(<pre existing test suites> <new-suite-name>)
+   ```
+
+With this in place, the new test suite should be able to be built and run:
+
+```bash
```
Comment (flamingbear) [inline README.md:107]: You have to set the env and this won't let you test against local.  That should be spelled out probably.

Maybe describe creating the test suite?  with the environment and running the notebook against the local HIAB.

**Review by flamingbear — CHANGES_REQUESTED** [review]
Two things that could be cleaner for new users.

File: README.md (hunk 1 above)
Comment (owenlittlejohns) [reply README.md:61]: Okay - I can make this statement more accurate (e.g., "you can't" rather than "it's not as good").

File: README.md (hunk 2 above)
Comment (owenlittlejohns) [reply README.md:107]: > You have to set the env and this won't let you test against local. That should be spelled out probably.

Good call, will do.

> Maybe describe creating the test suite? with the environment and running the notebook against the local HIAB.

Yeah - maybe I was a bit lazy here as some of this is covered in the section below. I'll improve this now.

Comment (flamingbear) [reply README.md:107]: You can always just point them to the correct location.

File: README.md (hunk 1 above)
Comment (owenlittlejohns) [reply README.md]: [Updated here](https://github.com/nasa/harmony-regression-tests/pull/79/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R58-R61)

File: README.md (hunk 2 above)
Comment (owenlittlejohns) [reply README.md:109]: Okay - I think that's updated now. I added a reference to HIAB, and also went into more detail on the contents of the subdirectory for each test suite.

**Review by flamingbear — APPROVED** [review]

**Comment by flamingbear** [comment]
New changes look good to me, thanks.

**Pull request merged**